index.html

```diff
@@ -39,6 +39,197 @@ <h1 class="greeting">I'm Ivan<br>Glad to meet <br>you!</h1>
       </button>
     </div>
   </div>
+  
+  <section class="work">
+    <div class="tonic-1">
+
+      <div class="tonic-img"></div>
+    <div class="text-box">
+      <div class="tonic-text">
+        <h1 class="tonic-title">Tonic</h1>
+
+        <ul class="info">
+          <li id="client">CANOPY</li>
+          <li id="role">Full stack Dev</li>
+          <li id="date"><span id="datetime"></span></li>
+          <script>
+            var dt = new Date();
+            document.getElementById("datetime").innerHTML = dt.getFullYear();
+          </script>
+        </ul>
```

Comment:
- [ ] You did great by using javascript but kindly remove the script pas at this level javascript is not required in the project you will use it later with a better understanding of how it works :+1:

Reply:
Fixed.